packages/core/src/hooks/types.ts

```diff
@@ -16,6 +16,7 @@ import type {
   HookToolConfig,
 } from './hookTranslator.js';
 import { defaultHookTranslator } from './hookTranslator.js';
+import type { ConfigSource } from './hookRegistry.js';
```

Comment:
Circular Dependency: ConfigSource is defined in hookRegistry.ts, but hookRegistry.ts imports types.ts. types.ts now imports ConfigSource back from hookRegistry.ts. it's cleaner to move the ConfigSource enum definition into types.ts to break the cycle